resources/views/pages/registration.twig

```diff
@@ -239,6 +239,14 @@
                                 'required': isTShirtSizeRequired,
                                 'required_icon': isTShirtSizeRequired,
                                 'selected': f.formData('tshirt_size', ''),
+                                'info': __('settings.profile.shirt_size.hint'),
+                                'raw_form_text': true,
+                                'form_text': (tShirtLink ? m.icon('info-circle')
+                                    ~ ' <a href="'
+                                    ~ tShirtLink|escape('html_attr')
+                                    ~ '">'
```

Comment:
A `target="_blank" rel="noopener"` is needed here (and on the profile settings) to not discard the form content (the noopener part should not be needed nowadays but :shrug:)

Reply:
done